docs/subgraph-manifest.md

```diff
@@ -74,3 +75,32 @@ When using the Graph-CLI, local paths may be used during development, and then,
 | Field | Type | Description |
 | --- | --- | --- |
 | **path** | *String or [IPLD Link](https://github.com/ipld/specs/)* | A path to a local file or IPLD link. |
+
+## 1.7 Dynamic Data Source
+A dynamic data source has all of the fields of a normal data source, except the [EthereumContractSource](#151-ethereumcontractsource) address field is removed.
```

Comment:
Instead of "except the address field is removed", let's phrase it "except it does not include a contract address under `source`. The address is a parameter that can later be provided when creating a dynamic data source from the template."